app/controllers/planning_application/cil_liability_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class PlanningApplication
+  class CilLiabilityController < ApplicationController
+    include PlanningApplicationAssessable
+    before_action :set_planning_application
+    before_action :ensure_planning_application_is_validated
+
+    def new
+      warn "in new"
```

Comment:
Do we need this warn?

Reply:
This was due to me trying to understand `config/routes.rb` 😅